libafl_frida/src/asan/asan_rt.rs

```diff
@@ -190,33 +190,22 @@ impl FridaRuntime for AsanRuntime {
         self.deregister_hooks(gum);
     }
 
-    fn pre_exec<I: libafl::inputs::Input + libafl::inputs::HasTargetBytes>(
-        &mut self,
-        input: &I,
-    ) -> Result<(), libafl::Error> {
-        let target_bytes = input.target_bytes();
-        let slice = target_bytes.as_slice();
-
-        self.unpoison(slice.as_ptr() as usize, slice.len());
+    fn pre_exec(&mut self, input_bytes: &[u8]) -> Result<(), libafl::Error> {
+        self.unpoison(input_bytes.as_ptr() as usize, input_bytes.len());
```

Comment:
Technically unpoison shoudln't use usize here lol, but I gues it should work on .. most? ... platforms
Not an issue with this PR of course.